package.json

```diff
@@ -19,6 +19,7 @@
     "node": ">=7.10.1"
   },
   "scripts": {
-    "start": "swagger-combine"
+    "start": "swagger-combine",
```

Comment:
lol sorry missed this earlier, typically `start` is reserved for spinning up web servers. Nothing wrong here, just a little off-convention :P